x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/pagerduty.tsx

```diff
@@ -175,20 +175,20 @@ const PagerDutyParamsFields: React.FunctionComponent<ActionParamsProps<PagerDuty
   } = actionParams;
   const severityOptions = [
     {
```

Comment:
The changes below are a re-ordering of the critical/error/warning/info severities, which changes the order displayed in the UI to match the same ordering I've seen on PD's site.